src/java/org/apache/cassandra/index/sai/plan/StorageAttachedIndexSearcher.java

```diff
@@ -153,11 +153,6 @@ private ResultRetriever(QueryController queryController,
         @Override
         public UnfilteredRowIterator computeNext()
         {
-            // IMPORTANT: The correctness of the entire query pipeline relies on the fact that we consume a token
-            // and materialize its keys before moving on to the next token in the flow. This sequence must not be broken
-            // with toList() or similar. (Both the union and intersection flow constructs, to avoid excessive object
-            // allocation, reuse their token mergers as they process individual positions on the ring.)
-
```

Comment:
This comment was redundant because it applied to different implementation of SAI that was using reactive flows and the sequencing of calls was very important.